userguide/content/en/docs/Adding content/navigation.md

```diff
@@ -210,11 +214,30 @@ If you are accepted to the program, you will receive the JavaScript code to add
 
 3. Disable the sidebar search in `config.toml` as this is not currently supported for Algolia DocSearch. See [Disabling the sidebar search box](#disabling-the-sidebar-search-box).
 
-3. Add the JavaScript code provided to you by Algolia to the head and body of every page on your site. See [Add code to head or before body end](/docs/adding-content/lookandfeel/#add-code-to-head-or-before-body-end) for details.
+3. Add the CSS and JS to use Algolia to the head and body of every page in your site, following the instructions in [Add code to head or before body end](/docs/adding-content/lookandfeel/#add-code-to-head-or-before-body-end).
+   
+   * In `head-end.html` add the DocSearch CSS:
+
+      ```html
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" />
+      ```
+   
```

Comment:
Come to think of it, why not just move this into `head-end.html`, relieving projects from doing one manual step?

Reply:
Hmm, I did it this way as that's how we did it in the previous integration, it's always been basically an empty file - also not sure if users would want a random CSS include on all their pages if they're not using Algolia. Thoughts?

Reply:
Maybe the file _should_ be empty, but it currently isn't: it contains logic that tests whether Algolia is configured. (We can address this through a separate PR.)

Reply:
Gotcha.